src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -128,6 +128,12 @@ public class GitHubSCMNavigator extends SCMNavigator {
     @CheckForNull
     private String credentialsId;
     /** The behavioural traits to apply. */
+
+    /**
+     * Whether to enable the retrieval of the Organization avatar. If false, then the default GitHub logo will be used.
+     */
+    private boolean enableAvatar;
```

Comment:
I think you would want to make this a `Boolean` and `readResolve` it to `true` (or treat `null` as `true` in the getter), to avoid changing behavior on existing org folders after upgrade?